packages/mdc-auto-init/package.json

```diff
@@ -3,6 +3,9 @@
   "description": "Declarative, easy-to-use auto-initialization for Material Components for the web",
   "version": "0.41.0",
   "main": "dist/mdc.autoInit.js",
+  "types": "dist/mdc.autoInit.d.ts",
```

Comment:
Make the order of these 3 added properties consistent across files (they seem shuffled in a few of them, in most they were added as modules, sideEffects, then types)